codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/message/XMLMessageUnmarshallableGenerator.kt

```diff
@@ -1,206 +0,0 @@
-package software.amazon.smithy.aws.swift.codegen.message
-
-import software.amazon.smithy.aws.swift.codegen.AWSClientRuntimeTypes
-import software.amazon.smithy.aws.swift.codegen.AWSSwiftDependency
-import software.amazon.smithy.codegen.core.CodegenException
-import software.amazon.smithy.codegen.core.Symbol
-import software.amazon.smithy.model.shapes.MemberShape
-import software.amazon.smithy.model.shapes.ShapeType
-import software.amazon.smithy.model.shapes.UnionShape
-import software.amazon.smithy.model.traits.EventHeaderTrait
-import software.amazon.smithy.model.traits.EventPayloadTrait
-import software.amazon.smithy.swift.codegen.ClientRuntimeTypes
-import software.amazon.smithy.swift.codegen.SwiftDependency
-import software.amazon.smithy.swift.codegen.SwiftTypes
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.serde.readwrite.DocumentReadingClosureUtils
-import software.amazon.smithy.swift.codegen.integration.serde.readwrite.ReadingClosureUtils
-import software.amazon.smithy.swift.codegen.model.eventStreamErrors
-import software.amazon.smithy.swift.codegen.model.eventStreamEvents
-import software.amazon.smithy.swift.codegen.model.expectShape
-import software.amazon.smithy.swift.codegen.model.hasTrait
-
-class XMLMessageUnmarshallableGenerator(val ctx: ProtocolGenerator.GenerationContext) {
```

Comment:
See `MessageUnmarshallableGenerator` above.  Don't know why Git didn't track the rename.